src/components/modal/base/modal.jsx

```diff
@@ -26,7 +26,10 @@ class Modal extends React.Component {
         return (
             <ReactModal
                 appElement={document.getElementById('app')}
-                bodyOpenClassName={this.props.useStandardSizes ? classNames('overflow-hidden') : null}
+                // only set bodyOpenClassName prop if this.props.useStandardSizes is true.
+                // (careful -- setting bodyOpenClassName to null here causes an error, because ReactModal
+                // does not correctly handle the case of a null bodyOpenClassName)
+                {...(this.props.useStandardSizes ? {bodyOpenClassName: classNames('overflow-hidden')} : {})}
```

Comment:
Could you define this object as a variable above the return statement? There doesn't seem to be a great solution but I think that'll be a little more readable.